### DISCUSSION TITLE
Yarn Start not working.

### DISCUSSION THREAD

**Discussion by jkiyak** [near/near-wallet#2115] [Q&A] [answered]
On the README.md, it says to build locally, you do yarn and then do yarn start. When I put in yarn start on the terminal, it just says command start not found. Am I doing something wrong? I'm trying to start making some changes but not sure where to start in terms of seeing them and such.

Edit: Okay so I found out that I wasn't doing the command in the right place, but when I go and do it in the right place this time (frontend), I get a different error message:

![image](https://user-images.githubusercontent.com/19947169/133537617-607db184-6290-4efc-a5b8-5d585c5e4af4.png)

**Comment by roshaans** [comment, marked as answer]
Yep, you should run `yarn start` in `packages/frontend/` folder
